index.md

```diff
@@ -13,80 +13,304 @@ The pre-requisites for RTVS are:
+editor, courtesy of the [SQL Server Data Tools for Visual Studio
+extension](https://msdn.microsoft.com/en-us/mt186501.aspx) that gives you syntax
+coloring and IntelliSense for your SQL code. 
+
+![SQL Window Query Results](./media/sql-query-results.png)
+
+Once you're happy with your SQL and your R Code, RTVS will handle combining your
+results and packaging them up to send to SQL Server. For more details, see the
+detailed walkthrough in our documentation for [SQL Server
+support](./sqlserver.html).
+
+### Scalable compute resources
+
+R developers and data scientists often prefer to use laptops in their day-to-day
+work. They develop their models against down-sampled datasets until they are
+satisfied with the results that they can get from those datasets. When they want
+to validate their models against larger datasets, they often have to move their
+code or their development environments to a more powerful machine.
+
+The [workspaces](./remote-execution.html) feature of RTVS makes it easy to bind
+RTVS to different workspaces. Local workspaces are simply other versions of the
+R interpreter that you have on your local computer. **Remote workspaces** are R
+interpreters that are installed on remote computers. By [installing RTVS Remote
+Services](./setup-remote.html) on a machine with lots of CPU cores, lots of RAM,
+lots of storage, or lots of GPU, you can tailor the execution environment to the
+needs of your code and the data that you want to analyze. A remote workspace can
+be shared by several team members, making it easier to collaborate on large or
+sensitive datasets that must reside on an IT-managed computer.
+
+![](./media/workspace-window.png)
+
+You can even setup your remote workspaces on [Azure virtual
+machines](https://azure.microsoft.com/en-us/services/virtual-machines/?b=17.08),
+eliminating the need for capital expenditures to get the computing resources
+that you need to get your work done. 
+
+When you are using a remote workspace, all editing and visualization are done
+using RTVS on your local computer. All code execution occurs on the remote
+computer. This ensures that the computing resources on your remote computer are
+totally dedicated to running your R code, without the overhead of running an IDE. 
+
+For more details, see the [workspaces documentation](./remote-execution.html).
 
 ### Table of contents for the documentation
 
-* [Variable Explorer](variable-explorer.html)
 * [Editor](editing.html)
+* [IntelliSense](intellisense.html)
 * [Code Navigation](code-navigation.html)
 * [Code Snippets](code-snippets.html)
+* [R Interactive Window](interactive-repl.html)
 * [History window](history.html)
-* [Help](help.html)
+* [Variable Explorer](variable-explorer.html)
 * [Plotting](plotting.html)
-* [SQL tooling](sqlserver.html)
-* [R Interactive Window](interactive-repl.html)
-* [IntelliSense](intellisense.html)
 * [Debugging](debugging.html)
```

Comment:
Debugging needs some updates. It still says that F5 and datatips are not supported (but will be in the future),  mentions a long-fixed bug about breakpoints in functions, and generally talks about "preview release".